Source/Hurl.Selector/Pages/SelectorPage.xaml

```diff
@@ -0,0 +1,165 @@
+                <UniformGridLayout
+                    ItemsJustification="Center"
+                    MaximumRowsOrColumns="4"
+                    MinColumnSpacing="12"
+                    MinItemHeight="120"
+                    MinItemWidth="120"
+                    MinRowSpacing="8" />
+            </ItemsRepeater.Layout>
+
+            <ItemsRepeater.ItemTemplate>
+                <DataTemplate x:DataType="models:Browser">
+                    <Grid>
+                        <Button
+                            Padding="8"
+                            HorizontalAlignment="Stretch"
+                            Command="{Binding ViewModel.LaunchBrowserCommand}"
+                            CommandParameter="{x:Bind}">
+                            <Grid RowSpacing="8">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="78" />
+                                    <RowDefinition />
+                                </Grid.RowDefinitions>
+                                <Image Source="{x:Bind Icon, Mode=OneWay}" />
+                                <TextBlock
+                                    Grid.Row="1"
+                                    FontSize="10"
+                                    Text="{x:Bind Name, Mode=OneWay}"
+                                    TextAlignment="Center"
+                                    TextWrapping="NoWrap" />
+                            </Grid>
+                            <Button.ContextFlyout>
+                                <MenuFlyout>
+                                    <MenuFlyoutItem Text="Reset" />
+                                </MenuFlyout>
+                            </Button.ContextFlyout>
+                        </Button>
+                        <Button
+                            x:Name="AdditionalBtn"
+                            Margin="2,3"
+                            Padding="2"
+                            HorizontalAlignment="Right"
+                            VerticalAlignment="Top"
+                            Background="{x:Null}"
+                            BorderBrush="{x:Null}"
+                            Visibility="{Binding ShowAdditionalBtn}">
+                            <Button.Content>
+                                <FontIcon Glyph="&#xE712;" />
+                            </Button.Content>
+                            <Button.Flyout>
+                                <MenuFlyout>
+                                    <MenuFlyoutItem Text="Reset" />
+                                </MenuFlyout>
+                            </Button.Flyout>
+                        </Button>
+                    </Grid>
+                </DataTemplate>
+            </ItemsRepeater.ItemTemplate>
+        </ItemsRepeater>
+
+        <Border
+            Grid.Row="2"
+            Padding="4"
+            HorizontalAlignment="Center"
+            Background="{StaticResource AccentAcrylicBackgroundFillColorBaseBrush}"
```

Comment:
Use a ThemeResource for anything that **could** change, depending on light or dark theme.